services/app-web/src/pages/GraphQLExplorer/GraphQLExplorer.tsx

```diff
@@ -0,0 +1,144 @@
+import { ApolloExplorer } from '@apollo/explorer/react'
+import styles from './GraphQLExplorer.module.scss'
+import { useQuery, gql } from '@apollo/client'
+import { Loading } from '../../components'
+import { useAuth } from '../../contexts/AuthContext'
+import { GenericErrorPage } from '../Errors/GenericErrorPage'
+
+const INTROSPECTION_QUERY = gql`
+    query IntrospectionQuery {
+        __schema {
+            queryType {
+                name
+            }
+            mutationType {
+                name
+            }
+            subscriptionType {
+                name
+            }
+            types {
+                ...FullType
+            }
+            directives {
+                name
+                description
+
+                locations
+                args {
+                    ...InputValue
+                }
+            }
+        }
+    }
+
+    fragment FullType on __Type {
+        kind
+        name
+        description
+
+        fields(includeDeprecated: true) {
+            name
+            description
+            args {
+                ...InputValue
+            }
+            type {
+                ...TypeRef
+            }
+            isDeprecated
+            deprecationReason
+        }
+        inputFields {
+            ...InputValue
+        }
+        interfaces {
+            ...TypeRef
+        }
+        enumValues(includeDeprecated: true) {
+            name
+            description
+            isDeprecated
+            deprecationReason
+        }
+        possibleTypes {
+            ...TypeRef
+        }
+    }
+
+    fragment InputValue on __InputValue {
+        name
+        description
+        type {
+            ...TypeRef
+        }
+        defaultValue
+    }
+
+    fragment TypeRef on __Type {
+        kind
+        name
+        ofType {
+            kind
+            name
+            ofType {
+                kind
+                name
+                ofType {
+                    kind
+                    name
+                    ofType {
+                        kind
+                        name
+                        ofType {
+                            kind
+                            name
+                            ofType {
+                                kind
+                                name
+                                ofType {
+                                    kind
+                                    name
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+`
+
+export const GraphQLExplorer = () => {
+    const endpointUrl = 'http://localhost:3030/local/graphql'
```

Comment:
is this supposed to be hard coded to local? I was understanding that we were going to embed into dev/val so we could test dev/val api

Reply:
Good catch, forgot about this one.